draft-ietf-quic-recovery.md

```diff
@@ -1003,15 +1003,21 @@ packets might cause the sender's bytes in flight to exceed the congestion window
 until an acknowledgement is received that establishes loss or delivery of
 packets.
 
-When an ACK frame is received that establishes loss of all in-flight packets
-sent prior to a threshold number of consecutive PTOs (pto_count is more than
-kPersistentCongestionThreshold, see {{cc-consts-of-interest}}), the network is
-considered to be experiencing persistent congestion, and the sender's congestion
-window MUST be reduced to the minimum congestion window (kMinimumWindow).  This
-response of collapsing the congestion window on persistent congestion is
-functionally similar to a sender's response on a Retransmission Timeout (RTO) in
-TCP {{RFC5681}}.
-
+## Persistent Congestion
+
+The network is considered to be experiencing persistent congestion when the
```

Comment:
The old text was more correct than this text in the sense that it's important that not only does time pass, but all the packets are lost within that period of time.

How about:
"When an ACK frame is received that establishes loss of all in-flight packets sent over a long enough period of time, the network is considered to be experiencing persistent congestion."

Reply:
For your suggestion, do you really need the word "all" in there. To me, the loss of any in-flight packets over a long enough time seems to be the requirement (at least how we have the pseudocode). It think the removal the word "all" might be better. Other than that, I am fine with making the change.

Reply:
It has to be all packets lost within that period of time(ie: total blackout).  And now that you mention it, that means the pseudocode isn't quite right either.

As #2255 details, the goal is to be consistent with the previous behavior in terms of when persistent congestion would be established.

Reply:
I'm not sure how we'd update the pseudocode to indicate loss of all packets. I'd have to think a bit more about it.